applications/teletype/priv/templates/service_added.html

```diff
@@ -82,7 +82,7 @@ <h2 style="margin:0;padding:0;text-align:center;font-family:'Open Sans',sans-ser
                         <table role="presentation" aria-hidden="true" border="0" cellpadding="0" cellspacing="0" align="center" width="100%" style="max-width:560px;border:1px solid #dedede;border-top-style:none;">
                             <tr>
                                 <td align="right" bgcolor="#ffffff" colspan="2" style="padding:10px 20px;font-family:'Open Sans',sans-serif;color:#999999;border-right:1px solid #dedede;"><small>TOTAL</small></td>
-                                <td align="right" bgcolor="#ffffff" colspan="1" style="padding:10px 20px;font-family:'Open Sans',sans-serif;color:#555555">&dollar;{{invoice.total}}</td>
+                                <td align="right" bgcolor="#ffffff" colspan="1" style="padding:10px 20px;font-family:'Open Sans',sans-serif;color:#555555">{{invoice.total}} USD</td>
```

Comment:
I wonder if we should include a "currency" macro so people can override their default currency from "USD" to "EUR" or "YEN" or whatever.

Reply:
If kazoo services provides the currency in the payload yes, just like transaction (braintree).